AnnoyancesFilter/sections/popups.txt

```diff
@@ -129,6 +129,9 @@ xn----7sba7aachdbqfnhtigrl.xn--j1amh#$#div[id^="pum-"][data-popmake*="nastrojka-
 xn----7sba7aachdbqfnhtigrl.xn--j1amh#$#html { overflow: visible!important; }
 backblaze.com##.ju_Con
 honeysanime.com##.buruburu-bee
+postandcourier.com#$#div[view="offer"] { display: none!important; }
+postandcourier.com#$#.tp-modal-open { overflow: auto !important; }
```

Comment:
This `postandcourier.com#%#//scriptlet('remove-class', 'tp-modal-open', 'body')` will be better - please check.
I still see this popup.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/155708135-327dd2a3-94af-41e5-842c-ec90b63509af.png)
</details><br/>